### ISSUE TITLE
location claim doesn't fully specify heading and speed

### ISSUE THREAD

**Issue by laurencelundblade** [ietf-rats-wg/eat#83]
WGS84 seems only to cover latitude and longitude

**Comment by gmandyam** [comment]
Current text states "The location coordinate claims are consistent with the WGS84 coordinate system", which is in line with comparable standards such as https://www.w3.org/TR/geolocation-API/#coordinates_interface.  This does not mean that WGS84 defines exactly how implementations derive speed or altitude (WGS84 actually defines both geodetic and orthometric height).

Note also that different implementations may leverage WGS84 coordinates, but allow for runtime configuration for how altitude calculation is performed.  See https://docs.microsoft.com/en-us/uwp/api/windows.devices.geolocation.altitudereferencesystem?view=winrt-19041 as an example.

I would recommend that the EAT spec not be overly prescriptive in defining how derived values such as altitude and speed are actually calculated.

**Comment by laurencelundblade** [comment]
Corrected the title to say heading and speed, not altitude and speed, are the problem. There is no specification for heading and speed and no specification at all is a problem.

I think this should just use tag 103 which seems to be a good simple WGS84-based specification of lat, long, altitude and accuracy. 

Heading and speed aren't really location. They should be in a different claim called "travel" or such.

Heading and speed were included because Qualcomm's location API included them. They can still be included in a token, but as part of different claim.

**Comment by gmandyam** [comment]
>Heading and speed were included because Qualcomm's location API included them. They can still be included in a token, but as part of different claim

Actually that was not my intention when I wrote this section - these attributes were included to maintain compatibility with W3C geoloc API coordinates I/F (https://www.w3.org/TR/geolocation-API/#coordinates_interface), which dates back to 2008.  Maintaining compatibility with this interface has the additional benefit of allowing browsers to leverage existing geoloc. API permissions framework to suppress all geoloc-related information in the attestation if so required by the end user - see https://www.w3.org/TR/webauthn/#browser-permissions-framework-extensions for a description of a comparable example, the location extension in WebAuthn API.  

Note also that the W3C Geoloc API has been defined at a level of abstraction beyond platform-specific location enablers (for better or worse).  If we go back to 2008, this meant that the positions I/F was meant to be compatible with BREW IPosdet (maybe this is what you were referring to as the "Qualcomm location API"?) and JSR 179.   The compatibility is still maintained with more recent enablers such as Android Location Manager.  So keeping compatibility with the W3C coordinates I/F should allow for maximum leverage across platforms.

I would not recommend bifurcating this claim.

**Comment by laurencelundblade** [comment]
OK. Fixed this up with #85 
Thanks for the W3C reference.
I will merge this in a day or two as it seems straight forward and we are aligned.

Also, I started to implement tag 103, but have come to not like it because it allows *any number* format including things like decimal fractions. There is no need for anything but double and single precision float numbers here. Anything else just causes interoperability problems.
